api/descriptions/policies/vulnerability_host_put.md

```diff
@@ -1,33 +1,48 @@
-Updates all rules in your vulnerability policy for container images in a single shot.
-Updating all rules at the same time makes it possible to maintain strict ordering between rules.
+Updates the vulnerability policy for your host deployments.
+All rules in the policy are updated in a single shot.
+
+This endpoint maps to the policy table in **Defend > Vulnerabilities > Hosts** in the Console UI.
+
+
+### cURL Request
+
+The following cURL command overwrites all rules in your current policy with a new policy that has a single rule.
+
+```bash
+$ curl 'https://<CONSOLE>/api/v1/policies/vulnerability/host?project=<PROJECT_NAME>' \
+  -X PUT \
+  -u <USER> \
+  -H 'Content-Type: application/json' \
+  -d \
+'{
+   "rules":[
+      {
+         "name":"<RULE_NAME>",
+         "collections":[
+            {
+               "name":"<COLLECTION_NAME>",
+
+               ...
+            }
+         ],
+         "alertThreshold":{
+            "disabled":false,
+            "value":1
+         },
+         "blockThreshold":{
```

Comment:
@KICKdesigns The `blockThreshold` stanza doesn't do anything (it's silently ignored). Please remove, and revalidate the curl command.

Reply:
@iansk blockThreshold seems to appear in the GET response. Are we sure that this should be removed for the PUT?

Reply:
See changes. Let me know if you want any other changes. In general, how do I know which params are the "minimum" that is required for each endpoint if all are generally accepted when submitting a PUT?

Reply:
@KICKdesigns I realize that this might be a frustrating exercise to find the "minimum" params. Let's follow this recipe going forward:

Step 1: (You) Figure out the technical minimum (as you've been doing so far), and submit that in your PR.
Step 2: (You) I'll review it, and let you know if there's anything else we should also provide.

On the backed, we've essentially got a master template for the major policy types (vuln, compliance, and runtime). We reuse that template across resource types (hosts, containers, images, etc) and locations (CI pipeline, registry, etc). That means, in some situations, some fields don't make sense. Even though the backend might accept those non-sensical settings, it's silently ignored.

I'm hoping that engr can implement better input validation in the future. In the meantime, I'd like to cover some of this gap with docs. The other goal is to give customers a good template building block from which they can craft more complicated (and meaningful) payloads.

Reply:
@KICKdesigns I removed the `blockThreshold` stanza for you. Also `graceDays` isn't minimally required, so I took it out.